DancingGoat/Web.config

```diff
@@ -1,112 +1,114 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <connectionStrings>
-    <clear/>
+    <clear />
   </connectionStrings>
   <appSettings>
-    <add key="webpages:Version" value="3.0.0.0"/>
-    <add key="webpages:Enabled" value="false"/>
-    <add key="ClientValidationEnabled" value="true"/>
-    <add key="UnobtrusiveJavaScriptEnabled" value="true"/>
-    <add key="ResponsiveWidths" value="200,400,600,800,1000,1200,1400,1600,2000,4000"/>
-    <add key="DefaultProjectId" value="975bf280-fd91-488c-994c-2f04416e5ee3"/>
+    <add key="webpages:Version" value="3.0.0.0" />
+    <add key="webpages:Enabled" value="false" />
+    <add key="ClientValidationEnabled" value="true" />
+    <add key="UnobtrusiveJavaScriptEnabled" value="true" />
+    <add key="ResponsiveWidths" value="200,400,600,800,1000,1200,1400,1600,2000,4000" />
+    <add key="DefaultProjectId" value="975bf280-fd91-488c-994c-2f04416e5ee3" />
+    <add key="ProjectId" value="975bf280-fd91-488c-994c-2f04416e5ee3" />
```

Comment:
remove `<add key="ProjectId" value="975bf280-fd91-488c-994c-2f04416e5ee3" />` - it should be initialized by the setup wizard

Reply:
removed